docs/organizations/managing-security-and-risk.md

```diff
@@ -26,44 +26,97 @@ The dashboard includes five areas:
 
 Each of these areas displays a count of matching items and a **Review** button to access a filtered list displaying the matching items.
 
-![Security and Risk Management dashboard](images/security-risk-management-dashboard.png)
+![security and risk management dashboard](images/security-risk-management-dashboard.png)
 
 ## Item list
 
-The Security and Risk Management item list displays a list of items, filtered by status and sorted by due date ascending.
+The **Security and risk management items** page displays a list of items, filtered by status and sorted by due date ascending.
 
-To access the item list, access the [dashboard](#dashboard) and click the **Review** button under the area of interest, based on the desired filtering.
+To access the item list, access the [dashboard](#dashboard) and click the **Review** button in the area of interest, based on the desired filtering.
 
 When viewing the item list:
 
 -   To update the filtering criteria, click the **Status** or **Priority** drop-downs above the list.
 -   To find out more about an item, click its **Details** column. This redirects you to a filtered view of the **Repository issues** page, showing only the item of interest.
 
-![Security and Risk Management item list](images/security-risk-management-item-list.png)
+![Security and risk management items](images/security-risk-management-item-list.png)
+
+## Managing integrations
+
+Integrations let you detect security vulnerabilities across a number of platforms and display them all as security items on the unified item list.
+
+The following section explains how to integrate with each of the supported platforms.
+
+=== "Codacy"
+
+    Security and risk management always displays items opened after detecting Codacy security issues. This feature can't be deactivated.
+
+=== "Jira Cloud"
+
+    To install the Jira Cloud integration:
+
+    1.  Open your organization **Security and Risk** page and click the **Configure** button. This opens the configuration page.
+
+    1.  On the right-hand side of the page, under the Integrations list, find the Jira entry and click the **Install** button. This redirects you to Atlassian's website.
+
+        ![Security and risk management Jira Cloud integration](images/security-risk-management-integration-jira.png)
+
+    1.  On Atlassian's website, authorize Codacy to access your Atlassian account. Once successful, you're redirected back to Codacy.
+
+        !!! note
+            Use a Jira account with admin permissions when authorizing Codacy. This lets Codacy access all issues, since the integration inherits the permissions of the authorizing account.
+
+        Upon installation, Codacy automatically imports all Jira tickets that were created within the 90-day period preceding the integration and tagged with the label "security" and displays them on the item list, along with items from other sources.
```

Comment:
Thanks! I'll just mention "open" to avoid getting too detailed on Jira's status grouping. Also mentioning that the label can be case-insensitive; that's useful.